.changeset/few-suns-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+Adds "yarn dev" command to simplify local development.
```

Comment:
On create-app changes, we usually add a few lines to inform users how they can make the corresponding change to their local `packages/app`.